backend/lcfs/tests/transfer/test_status_changes.py

```diff
@@ -12,6 +13,13 @@ def transaction_repo(dbsession):
 def transfer_repo(dbsession):
     return TransferRepository(db=dbsession)
 
+@pytest.fixture
```

Comment:
this is a great addition. In our next dev session it would be worth discussing its use case.

Reply:
Of course! Thanks, Alex, for reviewing my PR. :pray: